### PR TITLE
removes spring cloud stream - now using rest interface to report micr…

### PR DESCRIPTION
**Pull request by esotrope**
…oservice

### CODE REVIEW

File: pdf-report-processor/build.gradle [hunk 1]
```diff
@@ -18,13 +18,11 @@ dependencies {
     compile 'org.springframework.cloud:spring-cloud-starter-aws'
     compile 'org.springframework.cloud:spring-cloud-starter-config'
     compile 'org.springframework.cloud:spring-cloud-starter-stream-rabbit'
```
Comment (wtritch) [inline pdf-report-processor/build.gradle]: Should we get rid of spring-cloud-starter-stream-rabbit as well?

File: webapp/build.gradle [hunk 2]
```diff
@@ -19,7 +19,6 @@ dependencies {
     compile 'org.springframework.boot:spring-boot-starter-web'
     compile 'org.springframework.cloud:spring-cloud-starter-config'
     compile 'org.springframework.cloud:spring-cloud-starter-stream-rabbit'
```
Comment (wtritch) [inline webapp/build.gradle]: likewise here, remove the rabbit stream impl?

File: pdf-report-processor/build.gradle (hunk 1 above)
Comment (esotrope) [inline pdf-report-processor/build.gradle]: I'll test it without and remove it if we can get along without it

**Comment by malaffoon** [comment]
Shouldn't this have affected the docker config?

**Pull request merged**